libweb3jsonrpc/UnixSocketServer.cpp

```diff
@@ -19,7 +19,7 @@ along with cpp-ethereum.  If not, see <http://www.gnu.org/licenses/>.
 *   Arkadiy Paronyan <arkadiy@ethdev.com>
 * @date 2015
 */
-#ifndef WIN32
+#if !defined(_WIN32)
```

Comment:
When it is defined it is defined to 1. So I think it is a bit better to expect it to be 1 than `#if defined(_WIN32)` to be true for `_WIN32` set to 0.